datafusion/physical-plan/src/render_tree.rs

```diff
@@ -31,7 +31,7 @@ use crate::{DisplayFormatType, ExecutionPlan};
 // TODO: It's never used.
 /// Represents a 2D coordinate in the rendered tree.
 /// Used to track positions of nodes and their connections.
-#[allow(dead_code)]
+#[expect(dead_code)]
```

Comment:
Can we place these on the fields themselves instead of the struct, to make it clear its the fields that are unused?